app/src/main/java/com/claraberriel/mvpparking/mvp/presenter/ReleasePresenter.java

```diff
@@ -0,0 +1,74 @@
+package com.claraberriel.mvpparking.mvp.presenter;
+
+import android.util.Log;
+
+import androidx.annotation.VisibleForTesting;
+
+import com.claraberriel.mvpparking.entities.Parking;
+import com.claraberriel.mvpparking.mvp.model.ReleaseModel;
+import com.claraberriel.mvpparking.mvp.view.ReleaseView;
+
+public class ReleasePresenter {
+
+    private final ReleaseModel releaseModel;
+    private final ReleaseView releaseView;
+
+    public ReleasePresenter(ReleaseModel releaseModel, ReleaseView releaseView) {
+        this.releaseModel = releaseModel;
+        this.releaseView = releaseView;
+    }
+
+    public boolean onRelease() {
+        try {
+            releaseModel.checkIfAnyReservationExists();
+            if (isParkingLotNumberValid() && isSecurityCodeValid()) {
```

Comment:
![image](https://user-images.githubusercontent.com/83037219/126367830-c95fa0c7-b60b-4e95-a60d-9156c20e1983.png)
I believe this business rule is not covered, please correct me if I'm wrong

Reply:
Alright, I'll work on that thanks